pkgs/development/node-packages/node-packages-v10.json

```diff
@@ -1,9 +1,142 @@
 [
-  "bower"
+  "alloy"
+, "asar"
+, "azure-cli"
+, "azure-functions-core-tools"
+, "bower"
+, "bower2nix"
+, "browserify"
+, "castnow"
+, "clean-css"
 , "coffee-script"
+, "coinmon"
+, "configurable-http-proxy"
+, "cordova"
+, "create-cycle-app"
+, "create-react-app"
+, "create-react-native-app"
+, "csslint"
+, "dat"
+, "dhcp"
+, "dnschain"
+, "docker-registry-server"
+, "elasticdump"
+, "elm-oracle"
+, "elm-test"
+, "emoj"
+, "eslint"
+, "eslint_d"
+, "emojione"
+, { "fast-cli": "1.x" }
+, "fetch-bower"
+, "forever"
+, "git-run"
+, "git-ssb"
+, "git-standup"
+, "graphql-cli"
 , "grunt-cli"
+, { "guifi-earth": "https://github.com/jmendeth/guifi-earth/tarball/f3ee96835fd4fb0e3e12fadbd2cb782770d64854 " }
+, "gulp"
+, "gulp-cli"
+, "hipache"
+, "htmlhint"
+, "html-minifier"
+, "http-server"
+, "ionic"
+, "ios-deploy"
+, "istanbul"
+, "imapnotify"
+, "jake"
+, "javascript-typescript-langserver"
+, "jayschema"
+, "jsdoc"
+, "jshint"
+, "json"
+, "js-beautify"
+, "jsonlint"
+, "jsontool"
+, "json-diff"
+, "json-refs"
+, "json-server"
+, "js-yaml"
+, "karma"
+, { "kibana-authentication-proxy": "git://github.com/fangli/kibana-authentication-proxy.git" }
+, "lcov-result-merger"
+, "leetcode-cli"
+, "lerna"
+, "less"
+, "less-plugin-clean-css"
+, "live-server"
+, "livedown"
+, "madoko"
+, "mathjax"
+, "meat"
+, "meguca"
+, "mocha"
+, "multi-file-swagger"
+, "nijs"
+, "node2nix"
 , "node-gyp"
 , "node-gyp-build"
+, "node-inspector"
 , "node-pre-gyp"
+, "nodemon"
+, "node-red"
+, { "node-uptime": "https://github.com/fzaninotto/uptime/tarball/1c65756575f90f563a752e2a22892ba2981c79b7" }
```

Comment:
Also seems old, dependencies might be outdated (insecure): https://github.com/fzaninotto/uptime

Reply:
cc @shlevy